app/views/provider_interface/organisations/settings.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= content_for :browser_title, 'Organisation settings' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Organisation settings</h1>
+    <ul class="govuk-list govuk-list--spaced">
+      <% if current_provider_user.authorisation.can_manage_users_for_at_least_one_provider? %>
+        <li>
+          <%= govuk_link_to t('page_titles.provider.users'), provider_interface_provider_users_path %>
+        </li>
+      <% end %>
+
+      <% if current_provider_user.authorisation.can_manage_organisations_for_at_least_one_setup_provider? %>
+        <li>
+          <%= govuk_link_to t('page_titles.provider.org_permissions'), '#not-linked-yet' %>
```

Comment:
I think we need to have this called `organisation` rather than `organisational`

Reply:
The link uses `page_titles.provider.org_permissions`, I'd prefer to stick to this and then update `org_permissions` after this goes live, making the change everywhere. What do you think?

Reply:
I think if we go with adding a new translation lets say `page_titles.provider.organisation_permissions` and we use it explicitly here, we should be ok. Im worried we'll have to unpick this before we go live, and since we are using new views I think we will be ok

Reply:
Make sense, added.